kornia/utils/misc.py

```diff
@@ -52,3 +56,58 @@ def vec_like(n: int, tensor: Tensor, shared_memory: bool = False) -> Tensor:
 
     vec = zeros(n, 1, device=tensor.device, dtype=tensor.dtype)
     return vec[None].expand(tensor.shape[0], n, 1) if shared_memory else vec[None].repeat(tensor.shape[0], 1, 1)
+
+
+def differentiable_polynomial_rounding(input: Tensor) -> Tensor:
+    """This function implements differentiable rounding.
+
+    Args:
+        input (Tensor): Input tensor of any shape to be rounded.
+
+    Returns:
+        output (Tensor): Pseudo rounded tensor of the same shape as input tensor.
+    """
+    # Perform differentiable rounding
+    output: Tensor = torch.round(input) + (input - torch.round(input)) ** 3
```

Comment:
```suggestion
    x_round = x.round()
    output: Tensor = x_round + (x - x_round) ** 3
```
More idiomatic, one less torch import nd one less op